templates.go

```diff
@@ -66,42 +66,44 @@ import (
 	"google.golang.org/grpc/test/bufconn"
 )
 {{range $lis := .Listeners}}
-var (
-	// {{$lis}} is a global in-memory buffer that listeners that is
+type {{$lis | UpperCase }} struct {
+	// listener is a global in-memory buffer listener that is
```

Comment:
update comment as it is no longer global

Reply:
Good catch, fixed